catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -828,7 +828,7 @@ QuerySources initializeSources(
               if (!canAccessSource) {
                 notPermittedSources.add(source.getId());
               }
-              if (source.isAvailable() && canAccessSource) {
```

Comment:
This does lead to a bit of log pollution, at least for the source I was using (`CSW Specification Profile Federated Source`). As the query fails (which is should, the source is unavailable), the logs show `Caused by: javax.xml.bind.UnmarshalException` on each failed query.

Reply:
Probably means we just shouldn't log whatevers being logged at info/error in that csw source